am-lib/build.gradle

```diff
@@ -88,7 +96,7 @@ publishing {
       artifact sourcesJar
       artifact javadocJar
       groupId project.group
-      artifactId 'amlib'
+      artifactId 'am-lib'
```

Comment:
Is this change reflected in AM-API's dependencies?

Reply:
yep